src/graph_ir/low_graph_ir.cc

```diff
@@ -0,0 +1,53 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-----------------------------------------------------------------------------
+
+#include "low_graph_ir.h"
+#include <string>
+#include <vector>
+#include <sstream>
+
+using namespace std;
```

Comment:
Usually not recommended, the `std` namespace contains a huge amount of stuff. Better to just explicitly qualify things from `std` with `std::`.